bin/bootstrap.php

```diff
@@ -23,7 +25,29 @@
     }
 }
 
-die(sprintf(
+$composer_json_path = $cwd . '/composer.json';
+if (file_exists($composer_json_path)) {
+    $contents = file_get_contents($composer_json_path);
+
+    $composer_settings = json_decode($contents, true);
+    if (!is_array($composer_settings)) {
+        fwrite(STDERR, "Failed to load '$composer_json_path'\n");
+        exit(1);
+    }
+
+    $vendor_path = $composer_settings['config']['vendor-dir'] ?? $cwd . '/vendor';
+    if (!is_dir($vendor_path)) {
+        fwrite(STDERR, "Please check if 'composer.phar install' was run already (expected to find '$vendor_path')\n");
+        exit(1);
+    }
+
+    /** @noinspection PhpIncludeInspection */
+    require $vendor_path . '/autoload.php';
+    return;
+}
+
```

Comment:
This might decopuled to `includeProjectsAutoload()` function, so it's clear what it does.